consul/tests/test.bal

```diff
@@ -54,7 +54,7 @@ function testGetService() {
         io:println(<string>serviceDetails.detail().message);
         test:assertFail(msg = <string>serviceDetails.detail().message);
     } else {
-        test:assertNotEquals(serviceDetails, null, msg = "Failed to call getService()");
+        test:assertNotEquals(serviceDetails.length(), -1, msg = "Failed to call getService()");
```

Comment:
Again, these tests also will never fail right? Isn't there anything else specific to each scenario we can test? Like a value in the array. 

For the time-being maybe we can check if the length is greater than 0? Please create an issue to improve tests though.